CHANGELOG.rst

```diff
@@ -14,6 +14,14 @@ Change Log
 Unreleased
 ~~~~~~~~~~
 
+[0.1.6] - 2017-06-02
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Added
+_____
+
+* Support for Django 1.10
+
```

Comment:
@snowcrshd Since you've restored a good practice of adding changelog records, could you please also cover 1.05 - 1.0.1 versions? Don't spend too much time on it - something like

    [0.1.1] - [0.1.5] - 2017-06-01
    ~~~~~~~~~~~~~~~~~~~~~
    * Commit message 1
    * Commit message 2
    * Commit message 3


@doctoryes FYI - changelog records were missing